update.sh

```diff
@@ -65,7 +65,7 @@ for version in "${versions[@]}"; do
 
 	for v in \
 		windows/windowsservercore-{ltsc2016,1809} \
-		alpine3.12 \
+		alpine{3.12,3.13} \
```

Comment:
I think we need a similar change in https://github.com/docker-library/julia/blob/54c776a946bd429d567b0cadaf2ce4a34103c221/generate-stackbrew-library.sh#L93 (but in the other order), and adjust `defaultAlpineVariant` to `alpine3.13` :sweat_smile: